app/services/players_service.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+from uuid import UUID
+
+from app.core.config import settings
+
+from .base_service import BaseService
+
+
+class PlayersService(BaseService):
+    def __init__(self) -> None:
+        """Init the service."""
+        super().__init__()
+        self.name = "players-service"
+        self._set_base_url(
+            True, settings.PLAYERS_SERVICE_HOST, settings.PLAYERS_SERVICE_PORT
+        )
+        if settings.PLAYERS_SERVICE_API_KEY:
+            self.set_base_headers({"x-api-key": settings.PLAYERS_SERVICE_API_KEY})
+
+    async def create_player(self, user_public_id: UUID, telegram_id: int | None) -> Any:
+        """Create player using players service."""
+        payload = {"user_public_id": str(user_public_id)}
+        if telegram_id:
+            payload["telegram_id"] = int(telegram_id)
```

Comment:
El `telegram_id` es requisito obligatorio para la creación del Jugador.
```suggestion
        payload = {"user_public_id": str(user_public_id),
                           "telegram_id": int(telegram_id)}
```
Además, se va a crear un US en MS Jugador para esta validación.

Reply:
El `telegram_id` no es requisito obligatorio actualmente como se puede ver [acá](https://github.com/TPII-PadelPals/players-service/blob/dev/app/models/player.py#L25). En todo caso, si se crea una US para modificar eso, debería impactar acá también. Cambiarlo en esta US haría que se agregue código que no funciona a este microservicio

Reply:
Sea crea US: https://tpii-padel-pals.atlassian.net/browse/SCRUM-257